support-frontend/assets/pages/contributions-landing/components/ContributionStateDs.jsx

```diff
@@ -0,0 +1,183 @@
+// @flow
```

Comment:
Does this need to be a separate file? Having all this duplicated code feels a bit risky as we have to remember to maintain both

Reply:
Good point, but hopefully we will only have one file after the test (hopefully the DS one)!?